### DISCUSSION TITLE
유니티로 월드 제작 질문합니다. (floor 크기 관련)

### DISCUSSION THREAD

**Discussion by yoonjy51** [naverz/zepeto-studio-kor#517] [4. General]
유니티로 월드 개발을 목표로 하고 있는 초보 개발자입니다.

기획 한 월드를 모델링 중에 있는데요. 
floor(바닥)의 크기가 가로 세로 50m라고 가정했을 때, 하나의 프랍(50mx50m 박스)으로 모델링을 해도 문제가 생기지 않는지 궁금합니다.

왜 이러한 질문을 드리냐면 빌드잇에서 공간 구성을 할 때 1m정도 되는 floor 프랍을 이어 붙혀서 하는 방식을 채택 했더라구요.
그래서 혹시 유니티로 개발시에 통으로 모델링하면 문제가 발생될 수 있을거 같아서 질문드립니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요~
거대한 프랍을 사용하는 것은 특별한 문제 없습니다.
50mx50m 박스는 성능에는 큰 문제 없으며,
개발 중에 고려해야 할 팁 정도는 캐릭터가 박스 안으로 들어가는 경우가 있다면 Backface Culling등에 의해 안쪽면이 렌더링 안되는 경우만 조심하시면 될 것 같습니다.
